test/Interop/Cxx/class/synthesized-initializers-silgen.swift

```diff
@@ -1,37 +1,26 @@
 // RUN: %target-swift-frontend -I %S/Inputs -enable-cxx-interop -emit-silgen %s | %FileCheck %s
 
 import SynthesizedInitializers
+import Constructors
```

Comment:
I'm going to merge this test with `constructors-silgen`. It doesn't make sense to have a "synthesized initializers" test when we never actually synthesize an initializer.